pkg/cloudprovider/aws/block_storage_adapter.go

```diff
@@ -64,22 +60,11 @@ func NewBlockStorageAdapter(region, availabilityZone string) (cloudprovider.Bloc
 		return nil, err
 	}
 
-	// validate the availabilityZone
 	var (
```

Comment:
No need for a `var` block any more - please move this down below into the `blockStorageAdapter` assignment directly.

Reply:
Makes sense. Will do.